dlt/destinations/impl/filesystem/configuration.py

```diff
@@ -25,6 +25,8 @@ class FilesystemDestinationClientConfiguration(FilesystemConfigurationWithLocalF
     """Maximum number of pipeline state files to keep; 0 or negative value disables cleanup."""
     always_refresh_views: bool = False
     """Always refresh table scanner views by setting the newest table metadata or globbing table files"""
+    legacy_compression_without_ext: bool = False
```

Comment:
Did we not decide in the discussion to not have this anymore? I think we always add this extension now.

Reply:
Ok, after rereading the discussion, I think we decided to keep this